dev/com.ibm.ws.security.audit.source/src/com/ibm/ws/security/audit/event/utils/ParameterUtils.java

```diff
@@ -34,19 +34,31 @@ static protected StringBuffer format(Object param) {
                             if (element.getClass().isArray()) {
                                 buf.append(format(element));
                             } else {
-                                buf.append(element.toString());
+                                buf.append(ToString(element));
                             }
                         }
```

Comment:
Should there be an else here to append "null" when the element in the array is null?

Reply:
I modified the code on the ejbcollaboratorimpl class (caller) to handle an empty string, since this ParameterUtils class has been used for another components.

Reply:
This was written because basically the original code before my refactoring was skipping null, but now I think you are right, it should log the null object instead of just skipping it. Let me modify the code to log null object. I'm thinking it would be more important to log the accurate audit information.